internal/config/main_test.go

```diff
@@ -25,12 +29,242 @@ file:
 			FileConfiguration: &FileConfiguration{
 				Directory: "/foo/bar",
 			},
-			LDAPConfiguration: nil,
+			LDAPConfiguration:     nil,
 			DatabaseConfiguration: nil,
 		}
 
 		if !cmp.Equal(got, want) {
-			t.Errorf("got: %+v, want: %+v",	got, want)
+			t.Errorf("got: %+v, want: %+v", got, want)
+		}
+	})
+}
+
+func TestConfigWizardGetConfigurationWithLDAPBindPassFile(t *testing.T) {
+	tempDir := t.TempDir()
+	passwordFile := filepath.Join(tempDir, "password.txt")
+	passwordContent := "secret_password"
+	err := os.WriteFile(passwordFile, []byte(passwordContent), 0600)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	sourceConfig := "../../configs/examples/ldap/config.yml"
+	destConfig := filepath.Join(tempDir, "config.yml")
+
+	configContent, err := os.ReadFile(sourceConfig)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	configContent = bytes.Replace(configContent, []byte("# bind_pass_file: /path/to/password/file"), []byte(fmt.Sprintf("bind_pass_file: %s", passwordFile)), 1)
+	configContent = bytes.Replace(configContent, []byte("bind_pass:"), []byte("# bind_pass:"), 1)
+
+	err = os.WriteFile(destConfig, configContent, 0600)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Again, I'd rather not write any files to the filesystem as a result of tests. Plus, I think it's unnecessary to read the config and then find/replace a bunch of stuff to get the desired test config - just supply the config as a YAML string variable (like [here](https://github.com/peeley/carpal/blob/e8ec467f097bed9ad1418bfce1c90da7b3913b68/internal/config/main_test.go#L12)).